symfony/webpack-encore-bundle/1.6/config/packages/test/webpack_encore.yaml

```diff
@@ -0,0 +1,2 @@
+webpack_encore:
+    strict_mode: false
```

Comment:
I'm not convinced by this. I mean, I'm really mixed - I'm not against it. I've never personally hit this issue - we always build our Encore assets in test (because we do a lot of browser testing), and I would appreciate a clear exception if we forgot vs just the page being very broken. The fact that the `strict_mode` option exists might be enough? It really depends on what "most" users will do.

Reply:
Now `strict_mode` is commented for tests, it will allow users to fast change it if they need it. WDYT?